tests/unit/tasks/intern.ts

```diff
@@ -46,15 +55,15 @@ registerSuite('tasks/intern', function() {
 		},
 
 		beforeEach() {
-			mockRun.reset();
+			sandbox.reset();
 			mockRun.resolves();
```

Comment:
If we made `mockRun = sandbox.stub().resolves()`, could we get buy with just calling `sandbox.resetHistory()` above (instead of `sandbox.reset()`)?

Reply:
There are a few other stubs I've set up that need to be reset every time